.github/workflows/build-binaries.yml

```diff
@@ -227,7 +131,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
-          asset_path: ./bin/dungeon-revealer-linux.zip
+          asset_path: ./bin/dungeon-revealer-ubuntu.zip
```

Comment:
the executable will still be named ubuntu, right?

Reply:
No, pkg outputs `dungeon-revealer-linux`.